ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp

```diff
@@ -1030,13 +1032,7 @@ void bind_unary_composite_float_with_default(py::module& module, const unary_ope
         Returns:
             ttnn.Tensor: the output tensor.
 
-        Supported dtypes and layouts:
-
-        +----------------------------+---------------------------------+-------------------+
-        |     Dtypes                 |         Layouts                 |     Ranks         |
-        +----------------------------+---------------------------------+-------------------+
-        |    BFLOAT16                |       TILE                      |      2, 3, 4      |
-        +----------------------------+---------------------------------+-------------------+
+        {5}
```

Comment:
Also, add the `Note` heading here and just pass the information alone from the function call.